addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -331,7 +363,7 @@ return core.Class.extend({
             if (typeof tip.run === "function") {
                 tip.run.call(tip.widget, action_helper);
             } else if (tip.run !== undefined) {
-                var m = tip.run.match(/^(click|text|drag_and_drop) *(?:\(? *["']?(.+)["']? *\)?)?$/);
+                var m = tip.run.match(/^([a-zA-Z0-9_]+) *(?:\(? *(.+?) *\)?)?$/);
```

Comment:
Don't need ['"] because we take all the text each time, and it's confusing if you want to match "text" and the result must contains ".
(.+?) is the smaller match pattern also the reset \)? does not work.